### PR TITLE
Deploy koji configuration as koji file

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: puppet/modules/slave/manifests/init.pp [hunk 1]
```diff
@@ -400,12 +400,12 @@
...
     source => 'puppet:///modules/slave/titorc',
   }
-  file { '/home/jenkins/.koji/katello-config':
+  file { '/home/jenkins/.koji/koji':
     ensure => file,
     mode   => '0644',
     owner  => 'jenkins',
     group  => 'jenkins',
-    source => 'puppet:///modules/slave/katello-config',
+    source => 'puppet:///modules/slave/koji',
```
Comment (ekohl) [inline puppet/modules/slave/manifests/init.pp]: If you change the source here, then you also need to rename the file

**Comment by ehelms** [comment]
This follows the documented model for users now.

File: puppet/modules/slave/manifests/init.pp [hunk 2]
```diff
@@ -400,7 +400,7 @@
     group  => 'jenkins',
     source => 'puppet:///modules/slave/titorc',
   }
-  file { '/home/jenkins/.koji/katello-config':
```
Comment (ekohl) [inline puppet/modules/slave/manifests/init.pp]: I'd like an ```ensure => absent``` on this.

File: puppet/modules/slave/manifests/init.pp [hunk 3]
```diff
@@ -435,6 +435,12 @@
     owner  => 'jenkins',
     group  => 'jenkins',
   }
+  file { '/home/jenkins/.bin/kkoji':
```
Comment (ekohl) [inline puppet/modules/slave/manifests/init.pp]: is that dot intended to be there?

**Comment by ehelms** [comment]
@ekohl Good catch! I updated the one script I found using koji-katello which I think covers it.

**Pull request merged**